frameworks/hdfs/alerts/HDFS_alert_rules.yml

```
@@ -0,0 +1,96 @@
+groups:
+  - name: hdfs_info
+    rules:
+      # Alert for any instance that has a median request latency > 1s.
+    - alert: APIHighRequestLatency
+      expr: api_http_request_latencies_second{quantile="0.5"} > 1
+      for: 10m
+      labels:
+        severity: info
+        service: hdfs
+      annotations:
+        summary: "High request latency on {{ $labels.instance }}"
+        description: "Request latency is greater then 1s. Please consider to check network and other health cluster metrics"
+
+  - name: hdfs_warning
+    rules:
+    - alert: CapacityRemainingGB
+      expr: sum by (service_name,task_name) ({__name__=~".*_FSNamesystem_CapacityRemainingGB"}) < 10
+      for: 10m
+      labels:
+       severity: warning
+       service: hdfs
+      annotations:
+       summary: "Low capacity remaining in GB on {{ $labels.instance }} for {{ $labels.service_name }} / {{ $labels.task_name }}"
+       description: "{{ $labels.instance }} / {{ $labels.service_name }} / {{ $labels.task_name }} has capacity remaining {{ $value }}GB"
+
+    - alert: MissingBlocks
+      expr: sum by(service_name,task_name) (rate({__name__=~".*_dfs_FSNamesystem_MissingBlocks"}[10m])) > 0
+      labels:
+        severity: warning
+        service: hdfs
+      annotations:
+       summary: "Missing blocks on {{ $labels.instance }} for {{ $labels.service_name }} / {{ $labels.task_name }}"
+       description: "MissingBlocks value has increased for the last 10 min, you can lose your data"
```

you're missing a `for: 10m` if this is to be true
```suggestion
       description: "The number of missing blocks have increased over the last 10 min. Please investigate as this means you could lose your data"
```